pkg/api/vlabs/const.go

```diff
@@ -47,6 +47,10 @@ const (
 	MinDiskSizeGB = 1
 	// MaxDiskSizeGB specifies the maximum attached disk size
 	MaxDiskSizeGB = 1024
+	// MinIPAddressCount specifies the minimum number of IP addresses per network interface
+	MinIPAddressCount = 1
+	// MaxIPAddressCount specifies the maximum number of IP addresses per network interface
+	MaxIPAddressCount = 256
```

Comment:
I don't think there is an actual upper limit enforced by Azure SDN stack. I thought putting some reasonable upper limit is a good idea so that customers don't consume all addresses and resources inside the VM by mistake. 256 seemed large enough to fit all cases, and small enough to not cause any out-of-resource issues.

256, not 250, because coders do it in hex :)